packages/mdc-top-app-bar/_variables.scss

```diff
@@ -28,3 +28,9 @@ $mdc-top-app-bar-mobile-title-left-padding: 12px;
 $mdc-top-app-bar-mobile-section-padding: 4px;
 
 $mdc-top-app-bar-icon-size: 24px;
+
+$mdc-top-app-bar-short-collapsed-border-radius: 10px;
+$mdc-top-app-bar-short-collapsed-width: 56px;
+$mdc-top-app-bar-short-collapsed-right-icon-padding: 12px;
+
+$mdc-top-app-bar-standard-easing: cubic-bezier(.4, 0, .2, 1);
```

Comment:
Pretty sure this matches one of the variables in `mdc-animation`; reuse that instead?